### DISCUSSION TITLE
How to use Cell Plugin Migrations

### DISCUSSION THREAD

**Discussion by jonskeen** [react-page/react-page#1325] [Q&A] [answered]
Is there any kind of documentation on how to use CellPlugin migrations? The only thing I see in the docs is a `todo` to add more info about them.

**Comment by macrozone** [comment, marked as answer]
take a look on this

https://github.com/react-page/react-page/blob/master/packages/plugins/content/slate/src/migrations/v002.ts

this is the slate plugin (rich-text editor) that defines a migration

in your cell-plugin definition, you can then add it to the migration array https://github.com/react-page/react-page/blob/master/packages/plugins/content/slate/src/index.tsx#L67

and bump the version there. I think since some time, plugins start implicitly with 1 (the slate one is older hand started with a 0.something version).

i recommend to just use integers (1,2,3, etc.) to define the version. if you need to migrate something, bump the version number

**Reply by jonskeen** [reply]
This is exactly what I needed to see, thanks!
